src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.cs

```diff
@@ -427,7 +427,17 @@ public override BoundNode VisitTypeOrInstanceInitializers(BoundTypeOrInstanceIni
             {
                 if (IsFieldOrPropertyInitializer(initializer))
                 {
-                    statements.Add(RewriteExpressionStatement((BoundExpressionStatement)initializer, suppressInstrumentation: true));
+                    if (initializer.Kind == BoundKind.Block)
+                    {
+                        var block = (BoundBlock)initializer; 
+                        statements.Add(block.Update(block.Locals, block.LocalFunctions, 
```

Comment:
Nit: unconventional indent

Reply:
> Nit: unconventional indent

Looks conventional to me. Arguments for calls are aligned.

---
In reply to: [174216698](https://github.com/dotnet/roslyn/pull/25441#discussion_r174216698) [](ancestors = 174216698)